uplink/src/main.rs

```diff
@@ -142,19 +140,17 @@ fn main() -> Result<(), Error> {
         .unwrap();
 
     rt.block_on(async {
-        let mut handles = JoinSet::new();
         for (app, cfg) in config.tcpapps.iter() {
             let tcpjson = TcpJson::new(app.to_owned(), cfg.clone(), bridge.clone()).await;
-            handles.spawn(async move {
+            tokio::task::spawn(async move {
                 if let Err(e) = tcpjson.start().await {
                     error!("App failed. Error = {:?}", e);
                 }
             });
         }
-
-        while let Some(Err(e)) = handles.join_next().await {
-            error!("App failed. Error = {:?}", e);
-        }
     });
-    Ok(())
+
+    loop {
+        thread::sleep(Duration::from_secs(100));
```

Comment:
This can be replaced with https://github.com/bytebeamio/uplink/pull/216/files#diff-9e9525bac677e5b2bc2454915fd73d962ad3bc24994cc34997513243322c1cbdR157